blocks.md

```diff
@@ -1319,10 +1316,17 @@ interval = 1800
 
 Key | Values | Required | Default
 ----|--------|----------|--------
-`bytes` | Whether to use bytes or bits in the display (true for bytes, false for bits). | No | `false`
```

Comment:
Need to document that this has been removed. Perhaps better if we show what the new way is as well.

Reply:
Let's document these breaking change in NEWS.md. I started it off with a couple of examples from other blocks.